sympy/matrices/expressions/tests/test_matexpr.py

```diff
@@ -347,3 +347,8 @@ def test_factor_expand():
     assert expr1 != expr2
     assert expand(expr1) == expr2
     assert factor(expr2) == expr1
+
+def test_issue_2749():
+    A = MatrixSymbol("A", 5, 2)
+    assert (A.T*A).I.as_explicit()==Matrix([[(A.T*A).I[0, 0], (A.T*A).I[0, 1]], \
```

Comment:
Some space around == would be good. (And maybe around *, it looks quite crammed now, but most of the time there are no spaces around *.)